packages/cli/lib/sandboxes.js

```diff
@@ -148,6 +177,7 @@ function pollSyncTaskStatus(accountId, taskId, syncStatusUrl) {
     'lead-flows': 'forms', // lead-flows are a subset of forms. We combine these in the UI as a single item, so we want to merge here for consistency.
   };
   const barInstances = {};
+  let progressCounter = {};
```

Comment:
Keep track of each types progress so we never hit 100% before sync type status is complete